pkg/testing/fixture.go

```diff
@@ -715,20 +715,14 @@ func (e *ExecErr) Unwrap() error {
 // determine if the returned AgentStatusOutput is empty or not.
 // It should work with any 8.6+ agent
 func (f *Fixture) ExecStatus(ctx context.Context, opts ...process.CmdOption) (AgentStatusOutput, error) {
-	out, err := f.Exec(ctx, []string{"status", "--output", "json"}, opts...)
+	out, _ := f.Exec(ctx, []string{"status", "--output", "json"}, opts...)
+
 	status := AgentStatusOutput{}
 	if uerr := json.Unmarshal(out, &status); uerr != nil {
```

Comment:
after parse you should verify that there is at least a `status.Message` set, if its empty then it parsed an empty JSON and that should be an error.

Reply:
okay.
I think we can reuse `IsZero()` https://github.com/elastic/elastic-agent/blob/03b5d31eeda94b2f3f7ab83bded45c03738fb5a7/pkg/testing/fixture.go#L1245-L1247

Reply:
Yes that should work.

Reply:
PTAL